src/test/java/chapter7/question3/Question3Test.java

```diff
@@ -0,0 +1,63 @@
+package chapter7.question3;
+
+import org.junit.jupiter.api.Test;
+import src.chapter7.question3.NumberDivisibility;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Question3Test {
+
+    @Test
+    public void getTags_whenNumberIsDivisibleBy2_shouldReturnEven() {
+        var n = 2;
+        var string = "even";
+        var newNumber = new NumberDivisibility(n);
+        assertTrue(newNumber.getTags().contains("even"));
+    }
+
+    @Test
+    public void getTags_whenNumberIsDivisibleBy3_shouldReturnThree() {
+        var n = 3;
+        var newNumber = new NumberDivisibility(n);
+        assertTrue(newNumber.getTags().contains("three"));
+    }
+
+    @Test
+    public void getTags_whenNumberIsDivisibleBy5_shouldReturnFive() {
+        var n = 5;
+        var newNumber = new NumberDivisibility(n);
+        assertTrue(newNumber.getTags().contains("five"));
+    }
+
+    @Test
+    public void getTags_whenNumberIsDivisibleBy2and3_shouldReturn23() {
+        var n = 6;
+        var newNumber = new NumberDivisibility(n);
+        ArrayList<String> stringArray = new ArrayList<>();
+        stringArray.add("even");
+        stringArray.add("23");
+        assertEquals(stringArray, newNumber.getTags());
```

Comment:
I think it is better to replace assertEquals with assertTrue and check if newNumber.getTags() contains "23".

Reply:
This part has been replaced.